typhon/objects/networking/endpoints.py

```diff
@@ -321,3 +346,12 @@ def makeTCP4ServerEndpoint(port):
     """
 
     return TCP4ServerEndpoint(unwrapInt(port))
+
+
+@runnable(RUN_1)
+def makeTCP6ServerEndpoint(port):
```

Comment:
You don't have to add a test; this one's a little tricky to test easily. Still, you should definitely write a short echo module, like https://gist.github.com/b91222dbec645e9c1c3f024476531cc5, to test.